content/03.results.md

```diff
@@ -3,17 +3,17 @@
+![**Overview of the OpenPBTA Project.** A, The Children's Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumor samples from 943 patients. To date, 38 cell lines were created from tumor tissue and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or Targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Processed data was released through the OpenPBTA GitHub project, through which contributors filed issues and developed code for analyses within docker. Additional analysis details can be found in **Figure {@fig:S1}**. Simultaneously, this manuscript was written in GitHub using Manubot. All analysis and manuscript code was reviewed and continuously integrated using CircleCI. C, Stacked bar plot summary of the number of biospecimens per phase of therapy per broad histology (GNG = ganglioglioma, LGG = low-grade glioma, DIPG = diffuse intrinsic pontine glioma, DMG = diffuse midline glioma, EPN = ependymoma, HGG = high-grade glioma, ATRT = atypical teratoid rhabdoid tumor, CNS-ET = CNS embryonal tumor, MB = medulloblastoma, PNF = plexiform neurofibroma, DNET = dysembryoplastic neuroepithelial tumor, CRANIO = craniopharyngioma, EWS = Ewing sarcoma, CPP = choroid plexus papilloma. Panel A created with BioRender.com](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/e68062fc6452287a5a2782d027055ce3e7e93d78/figures/pngs/figure1.png?sanitize=true){#fig:Fig1 width="7in"}
```

Comment:
Perhaps you tested this already:

```suggestion
![**Overview of the OpenPBTA Project.** A, The Children's Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumor samples from 943 patients. To date, 38 cell lines were created from tumor tissue and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or Targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Processed data was released through the OpenPBTA GitHub project, through which contributors filed issues and developed code for analyses within docker. Additional analysis details can be found in **Figure {@fig:S1}**. Simultaneously, this manuscript was written in GitHub using Manubot. All analysis and manuscript code was reviewed and continuously integrated using CircleCI. C, Stacked bar plot summary of the number of biospecimens per phase of therapy per broad histology (GNG = ganglioglioma, LGG = low-grade glioma, DIPG = diffuse intrinsic pontine glioma, DMG = diffuse midline glioma, EPN = ependymoma, HGG = high-grade glioma, ATRT = atypical teratoid rhabdoid tumor, CNS-ET = CNS embryonal tumor, MB = medulloblastoma, PNF = plexiform neurofibroma, DNET = dysembryoplastic neuroepithelial tumor, CRANIO = craniopharyngioma, EWS = Ewing sarcoma, CPP = choroid plexus papilloma. Panel A created with [BioRender.com](https://biorender.com/)](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/e68062fc6452287a5a2782d027055ce3e7e93d78/figures/pngs/figure1.png?sanitize=true){#fig:Fig1 width="7in"}
```

Reply:
update: this actually does work even though it doesn't appear it would in sublime text. pushed in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/244/commits/149329f186fc73f2c72812a329b97d465735c612

Reply:
OK, just kidding on this - it creates a hyperlink, but the url goes to `file:///converted/BioRender.com`, so that won't work. Sigh.